_config.yml

```diff
@@ -151,6 +157,17 @@ flink_statefun_releases:
           asc_url: "https://downloads.apache.org/flink/flink-statefun-3.1.1/flink-statefun-3.1.1-src.tgz.asc"
           sha512_url: "https://downloads.apache.org/flink/flink-statefun-3.1.1/flink-statefun-3.1.1-src.tgz.sha512"
 
+flink_ml_releases:
+  -
+      version_short: "2.0"
+      source_release:
+          name: "Apache Flink ML 2.0.0"
+          id: "200-ml-download-source"
+          flink_version: "1.14.2"
```

Comment:
Should this be 1.14.0? ( https://github.com/apache/flink-ml/blob/release-2.0/pom.xml#L73)

Reply:
Yes, indeed, I changed the version~